### PR TITLE
Document passthroughs [risk: low]

### PR DESCRIPTION
**Pull request by davidangb**
* in swagger yaml, define whether or not each api is a passthrough
* for those that are passthroughs,  define the target subservice to which we delegate
* some swagger yaml syntax fixes
* delete one passthrough whose target no longer exists
* fix the `examples` section for cromiam metadata api

@kcibul - does this give you what you need to define api testing strategy?
@aednichols - you okay with enabling the cromiam example?

-----

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/firecloud/service/EntityService.scala [hunk 1]
```diff
@@ -72,7 +72,7 @@ trait EntityService extends HttpService with PerRequestCreator with FireCloudDir
                   pathPrefix(Segment) { entityName =>
                     pathEnd {
                       requireUserInfo() { _ =>
-                        passthrough(entityTypeUrl + "/" + entityName, HttpMethods.GET, HttpMethods.PATCH, HttpMethods.DELETE)
+                        passthrough(entityTypeUrl + "/" + entityName, HttpMethods.GET, HttpMethods.PATCH)
```
Comment (davidangb) [inline src/main/scala/org/broadinstitute/dsde/firecloud/service/EntityService.scala:75]: the `DELETE` path is not documented in orch swagger, and does not exist in Rawls ... i.e. it's undocumented and it doesn't work. So I'm deleting it.

File: src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -692,133 +725,131 @@ paths:
...
+                "blah2.grep.pattern": "h",
+                "blah2.cat.asdf": "gs://cromwell-root/blah/c0c116ef-aac6-4029-8d0a-70d86b17c843/call-ls/job.stdout.txt",
+                "blah.grep.pattern": "v",
+                "blah.ls.pattern": "./"
+              },
+              "submission": "2015-12-11T16:53:21.000-05:00",
+              "status": "Succeeded",
+              "end": "2015-12-11T16:53:23.000-05:00",
+              "start": "2015-12-11T16:53:21.000-05:00"
+            }
```
Comment (davidangb) [inline src/main/resources/swagger/api-docs.yaml:852]: uncommented, and  moved from
```
responses:
  '200':
    examples:
      application/json:
```
to
```
responses:
  '200':
    examples:
```

File: src/main/resources/swagger/api-docs.yaml [hunk 3]
```diff
@@ -2318,45 +2427,47 @@ paths:
...
+        200:
+          description: Success
+          schema:
+            $ref: '#/definitions/TrialOperationResponse'
+        204:
+          description: Success (No Content)
+        403:
+          description: Forbidden; you must be a campaign manager to call this endpoint.
+        500:
+          description: Internal Server Error
```
Comment (davidangb) [inline src/main/resources/swagger/api-docs.yaml:2466]: changed indentation only

File: src/main/resources/swagger/api-docs.yaml [hunk 4]
```diff
@@ -692,133 +725,131 @@ paths:
...
-#                  "blah2.cat.asdf": "gs://cromwell-root/blah/c0c116ef-aac6-4029-8d0a-70d86b17c843/call-ls/job.stdout.txt",
-#                  "blah.grep.pattern": "v",
-#                  "blah.ls.pattern": "./"
-#                },
-#                "submission": "2015-12-11T16:53:21.000-05:00",
-#                "status": "Succeeded",
-#                "end": "2015-12-11T16:53:23.000-05:00",
-#                "start": "2015-12-11T16:53:21.000-05:00"
-#              }
+          examples:
```
Comment (aednichols) [inline src/main/resources/swagger/api-docs.yaml:728]: Thank you

**Review by aednichols — APPROVED** [review]
👍 Nice!

File: src/main/resources/swagger/api-docs.yaml [hunk 5]
```diff
@@ -532,6 +557,8 @@ paths:
 
   /api/workflows/{version}/{id}/abort:
     post:
+      x-passthrough: true
+      x-passthrouh-target: cromiam
```
Comment (jmthibault79) [inline src/main/resources/swagger/api-docs.yaml]: typo; repeated elsewhere

Comment (kcibul) [reply src/main/resources/swagger/api-docs.yaml]: 🦅 👀 nice catch

**Review by kcibul — APPROVED** [review]
This is fantastic -- great implementation idea.  I'm assuming all the content you have is correct, I didn't review that

**Comment by kcibul** [comment]
do these x- fields show up in the public swagger UI?  Not a dealbreaker if they do, just curious

**Comment by davidangb** [comment]
@kcibul no - the `x-` fields do not show up  in swagger-ui. I wish that  were an option!

**Pull request merged**